dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,64 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseInspectorOptions):
+
+    def __init__(self, is_enabled=True, is_case_sensitive=True,
+                 stop_words=None):
+        """
+        Constructs the TextProfilerOption object with default values.
+
+        :ivar is_enabled: boolean option to enable/disable the option.
+        :vartype is_enabled: bool
+        :ivar is_case_sensitive: option set for case sensitivity.
+        :vartype is_case_sensitive: bool
+        :ivar stop_words: option set for stop words.
+        :vartype stop_words: Union[None, list(str)]
+        :ivar words: option set for word update.
+        :vartype words: BooleanOption
+        :ivar vocab: option set for vocab update.
+        :vartype vocab: BooleanOption
+        """
+        self.is_case_sensitive = is_case_sensitive
+        self.stop_words = stop_words
+        self.words = BooleanOption(is_enabled=True)
+        self.vocab = BooleanOption(is_enabled=True)
+        super().__init__(is_enabled=is_enabled)
+
+    def _validate_helper(self, variable_path='TextProfilerOptions'):
+        """
+        Validates the options do not conflict and cause errors.
+
+        :param variable_path: current path to variable set.
+        :type variable_path: str
+        :return: list of errors (if raise_error is false)
+        :rtype: list(str)
+        """
+        if not variable_path:
+            variable_path = self.__class__.__name__
+
+        errors = super()._validate_helper(variable_path=variable_path)
+
+        if not isinstance(self.is_case_sensitive, bool):
+            errors.append("{}.is_case_sensitive must be a Boolean."
+                          .format(variable_path))
+
+        if self.stop_words is not None and \
+                (not isinstance(self.stop_words, list)
+                 or not all(
```

Comment:
not needed change, but it might be more readable if instead of breaking at the `all` we break at the for:
e.g.
```python
if (self.stop_words is not None and
        (not isinstance(self.stop_words, list)
         or not all(isinstance(item, str)
                    for item in self.stop_words))):
```
thoughts?

Reply:
looks better, changed it